src/ExpenseTrackerGrupo2/Persistence/Database/Scripts/init.sql

```diff
@@ -0,0 +1,76 @@
+CREATE TABLE IF NOT EXISTS users (
+    user_id UUID PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL UNIQUE,
+    password_hash VARCHAR(100) NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS expenses (
+    expense_id UUID PRIMARY KEY,
+    user_id UUID NOT NULL,
+    amount DECIMAL(10, 2) NOT NULL,
+    description VARCHAR(255),
+    category VARCHAR(100),
+    expense_date DATE NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (user_id) REFERENCES users(user_id)
```

Comment:
Don't forget to review the previous comment I left you for this part, which is relate to the delete type given the relationships we have. In this case, we have one to many and given that when a user is deleted, all the data should be deleted or not.

Reply:
for a removal its a sql protocol that it doesnt allow to delete if theres is any data left in other tables related to that object.

Reply:
Yeah, but I'm referring to delete referential action that we can use on the tables. Some of them are:
- `ON DELETE RESTRICT`
- `ON DELETE SET NULL`
- `ON DELETE SET DEFAULT`

And many more, review about them because what will happen with the data we have and I delete a user which has many expenses. The expenses will delete as well?

Reply:
I already explained it, in the comment above, but to be more specific, SQL lite, already set has default the "ON DELETE RESTRICT" so does not allow a referenced row in a parent table to be deleted if related rows exist in the child table. An error will be thrown if you try to delete the row.

Reply:
Is it the best option we can use?

Reply:
I think that its the more secure but already setted on cascade for a clean delete of the rows.